tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj

```diff
@@ -22,14 +27,14 @@
   <ItemGroup>
     <Compile Include="..\testproject\Common\TestResourceNames.cs" />
 
-    <None Include="..\testproject\**\*" Link="testassets\testproject\%(RecursiveDir)%(FileName)%(Extension)" CopyToOutputDirectory="PreserveNewest" />
-    <None Include="..\.editorconfig" Link="testassets\%(FileName)%(Extension)" CopyToOutputDirectory="PreserveNewest" />
-    <None Condition="'$(InstallWorkloadForTesting)' == 'true'" Include="$(PatchedNuGetConfigPath)" Link="testassets\testproject\nuget.config" CopyToOutputDirectory="PreserveNewest" />
-    <None Include="$(RepoRoot)Directory.Packages.props" Link="testassets\testproject\Directory.Packages.repo.props" CopyToOutputDirectory="PreserveNewest" />
-
     <PackageReference Include="Microsoft.Extensions.Http.Resilience" />
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(_BuildForTestsRunningOutsideOfRepo)' == 'true'">
+    <None Include="..\testproject\**\*" Link="testassets\testproject\%(RecursiveDir)%(FileName)%(Extension)" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="..\.editorconfig" Link="testassets\%(FileName)%(Extension)" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Should copying the tests\.editorconfig be part of the "infra"?

Reply:
This exists in `main`. IIRC, this was become some test seemed to indirectly depend on this file being available. So, we are copying it here as part of preparing the output dir to be usable for out-of-repo builds.

Reply:
And I didn't hit a need for this other projects that use out-of-repo (like future playground, and hosting.testing tests).